deploy/prometheus/templates/clusterversion.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   clusterDefinitionRef: prometheus
   componentVersions:
-  - componentDefRef: server
+  - componentDefRef: prometheus
```

Comment:
It also exists in deploy/prometheus/templates/backuppolicytemplate.yaml

Reply:
k